File: java/Fibonacci.java
public class Fibonacci {
    public static void main (String[] args) {
        for (int i = 0; i < 20; i++){
            System.out.print(fibonacci(i));
            System.out.print(" ");
        }
        System.out.print("\n");
    }

    static int fibonacci(int n) {
        assert n >= 0;

        if (n < 2) {
            return 1;
        }
        return fibonacci(n-1) + fibonacci(n-2);
    }

}

File: julia/fibonacci.jl
function fibonacci(n)
    if n < 1
        println("Error: n must be positive.")
    elseif n == 1 || n == 2
        return 1
    end
    return fibonacci(n-1) + fibonacci(n-2)
end


for i = 1:10
    print(fibonacci(i), " ")
end
print("\n")